CMakeLists.txt

```diff
@@ -342,7 +350,15 @@ IF((NOT OPENGL_BACKEND STREQUAL "None") OR USE_OPENGL)
 		ADD_DEFINITIONS("-DUSE_GL")
 		# Don't delete this - there is code that relies on -DUSE_OPENGL=1
 		SET(USE_OPENGL 1)
-		INCLUDE(FindGLEW)
+		IF(WIN32 AND _VCPKG_INSTALLED_DIR)
+			find_package(GLEW REQUIRED)
+			GET_TARGET_PROPERTY(GLEW_INCLUDE_DIR GLEW::GLEW INTERFACE_INCLUDE_DIRECTORIES)
+			GET_TARGET_PROPERTY(GLEW_LIBRARY_DEBUG GLEW::GLEW IMPORTED_IMPLIB_DEBUG)
+			GET_TARGET_PROPERTY(GLEW_LIBRARY_RELEASE GLEW::GLEW IMPORTED_IMPLIB_RELEASE)
+			SET(GLEW_LIBRARIES debug ${GLEW_LIBRARY_DEBUG} optimized ${GLEW_LIBRARY_RELEASE})
+		ELSE()
+			INCLUDE(FindGLEW)
+		ENDIF()
```

Comment:
Similarly here, except that cmake itself ships FindGLEW.

Reply:
IConv as well?

Reply:
Would require a cmake bump first. Not sure when it was added, but it's not in 3.1.:
https://cmake.org/cmake/help/v3.11/module/FindIconv.html

Reply:
The use of IConv is very limited and arguably not needed on Windows with its own built-in codepage API. Many Unix descendants have it inside their C runtime too. Seems like a weak candidate to bump CMake version up for.

Reply:
Agreed, especially since it looks like I nailed the version, which is from 2018.